app/Http/Controllers/V2/AuditStatus/GetAuditStatusController.php

```diff
@@ -0,0 +1,84 @@
+<?php
+
+namespace App\Http\Controllers\V2\AuditStatus;
+
+use App\Http\Controllers\Controller;
+use App\Http\Resources\V2\AuditStatusResource;
+use App\Models\V2\AuditStatus\AuditStatus;
+use App\Models\V2\Projects\Project;
+use App\Models\V2\Sites\Site;
+use App\Models\V2\Sites\SitePolygon;
+use Illuminate\Http\Request;
+
+class GetAuditStatusController extends Controller
+{
+    public function __invoke(Request $request, string $id = null)
```

Comment:
It looks like `$id` is not used below, and it doesn't appear to be part of the API definition. Please remove.